scripts/elasticsearch_upgrade.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## Shell Opts ----------------------------------------------------------------
+set -e -u -x
+set -o pipefail
+
+pushd /opt/rpc-openstack/rpcd/playbooks
+    # Update pip.conf on Elasticsearch container
+    openstack-ansible /opt/rpc-upgrades/playbooks/elasticsearch-postleap-pip-upgrade.yml
+
+    # Stop logstash service on Logstash container
+    openstack-ansible /opt/rpc-upgrades/playbooks/logstash-stop.yml
```

Comment:
Why using a playbook if you can just use ansible to stop the service ?

Reply:
Using ansible ad-hoc can achieve same goal, but these scripts were already passed tests. So optimization could be addressed later. For now top goal is merging this functionality.